ibacm/prov/acmp/src/acmp.c

```diff
@@ -2643,7 +2644,7 @@ static void acmp_port_up(struct acmp_port *port)
 	acmp_set_dest_addr(&port->sa_dest, ACM_ADDRESS_LID,
 			   (uint8_t *) &sm_lid, sizeof(sm_lid));
 
-	atomic_set(&port->sa_dest.refcnt, 1);
+	instance = atomic_inc(&port->sa_dest.refcnt) - 1;
```

Comment:
Why are systems with a VF adding the PF multiple times, that seems strange, maybe explain that in the commit message.

I don't know this code at all, so I'm not sure who will review this patch

@weiny2 do you care?

Reply:
From the commit message: "For a system with VFs enabled, the ports for both the PF and the VFs
are added."

Reply:
That doesn't explain why the VF and PF ports are intermingled in this algorithm..

Reply:
Well, the design of this algorithm is somewhat outside the scope of this bug fix. We discussed it on the mailing list, see https://www.spinics.net/lists/linux-rdma/msg69934.html

Bottom line, the endpoints are designated by the **Node** GUID, hence ports from both VFs and PFs are associated with the same struct _acmp_port_.

Reply:
Just sent a new version with more elaborate commit message on this point.